# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/tcp_wrappers/hotfix-clang.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- tcp_wrappers_7.6/myvsyslog.c.vanilla	2023-05-16 16:13:55.486744072 +0200
+++ tcp_wrappers_7.6/myvsyslog.c	2023-05-16 16:14:12.841743596 +0200
@@ -18,10 +18,7 @@
 #include "tcpd.h"
 #include "mystdarg.h"
 
-myvsyslog(severity, format, ap)
-int     severity;
-char   *format;
-va_list ap;
+void myvsyslog(int severity, const char* format, va_list ap)
 {
     char    fbuf[BUFSIZ];
     char    obuf[3 * STRING_LENGTH];
